superset-frontend/src/dashboard/components/SliceAdder.jsx

```diff
@@ -45,7 +45,6 @@ const propTypes = {
 };
 
 const defaultProps = {
-  selectedSliceIds: [],
```

Comment:
let's remove `.isRequired` here to (unless you can confirm you checked all the calls)

Reply:
I did, but you're right, let's keep the default and remove isRequired